src/main/java/org/gridsuite/geodata/server/GeoDataService.java

```diff
@@ -480,21 +480,35 @@ public List<LineGeoData> getLinesByCountries(Network network, Set<Country> count
 
         StopWatch stopWatch = StopWatch.createStarted();
 
-        List<Line> lines = network.getLineStream().collect(Collectors.toList());
-        List<HvdcLine> hvdcLines = network.getHvdcLineStream().collect(Collectors.toList());
+        List<Line> lines = network.getLineStream().toList();
+        List<TieLine> tieLines = network.getTieLineStream().toList();
```

Comment:
I see no reason to do a `toList()` here. If we kept this in the form of `Stream<TieLine>` everything would be simpler because in any case every time we use it below we turn it into a `stream()`.

Same for all the other lists.

Reply:
I agree that the function could be more optimized, but that is not the aim of this pull request.